src/main/java/bio/terra/cli/command/workspace/List.java

```diff
@@ -47,7 +47,16 @@ private void printText(java.util.List<WorkspaceDescription> returnValue) {
                   && workspaceContext.getWorkspaceId().equals(workspace.getId()))
               ? " * "
               : "   ";
-      System.out.println(prefix + workspace.getId());
+      OUT.println(prefix + workspace.getId());
+
+      String displayName = workspace.getDisplayName();
+      if (!(displayName == null || displayName.isBlank())) {
+        OUT.println("     Name: " + displayName);
```

Comment:
make a string constant for the indentation (or label format) so you keep things aligned).

Reply:
I took a hack at this, please take a look, it right justifies which looks a little better to me: 
```
jczerk-macbookpro:TestWorkspace jczerk$ terra workspace list
 * af5f3128-f0ff-4adf-9b35-c0052f710537
            Name: jczerkow_test
     Description: My Description
```